packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -76,7 +76,31 @@ class PageController extends ScrollController {
   final double viewportFraction;
 
   /// The current page displayed in the controlled [PageView].
+  ///
+  /// There are circumstances that this [PageController] can't know the current
+  /// page. Mainly when no [PageView]s are attached. Reading [page] will throw
+  /// an [AssertionError] in the following cases:
+  ///
+  /// 1. Zero [PageView] attached. On attaching or re-attaching, the new [page]
```

Comment:
The grammar of `Zero [PageView] attached.` is wrong. I'd probably fix it by saying "No [PageView]s are currently using the [PageController].` (This avoids the dubious implication that `PageView`s themselves get `attach()`ed, rather than the `_PagePosition`s)

Reply:
Done